README-CONAN.md

```diff
@@ -246,33 +514,54 @@ PROJECT imports(): Copied 5 '.dll' files
 
 Indicating that the packages were found in the local cache.
 
-That´s all. After a successful call to `conan install`, some files will be generated in the `build` folder that will
-allow to CMake (or other systems) to find the headers and libraries of the dependencies handled by conan. By simply
-running:
+[TOC](#TOC)
+
+<name id="4">
+## 4 Building Exiv2 with Adobe XMPsdk 2016
+
+With Exiv2 v0.27, you can build Exiv2 with Adobe XMPsdk 2016 on Linux/GCC, Mac/clang and Visual Studio 2017.
+Other platforms such as Cygwin are not supported by Adobe.  Adobe/XMPsdk is built as a external library.
+Applications which wish use the Adobe XMPsdk directly should build Exiv2 in this configuration and the
+library can be used by the application and Exiv2.  The Adobe XMPsdk can be built as a static or shared library (.DLL)
+
+To build Exiv2 with Adobe XMPsdk 2016, should perform steps 1.1, 1.2 and 1.3 described above, then perform the following:
```

Comment:
XMPsdk 2016, should perform => XMPsdk 2016, [you|the user] should perform

Reply:
I agree.    This is incorrect English.